test/auth.js

```diff
@@ -9,9 +9,9 @@ const Account = require("../src/models/Account");
 
 describe("Register endpoint test", () => {
 
-    afterEach("Clearing accounts", async () => {
```

Comment:
A problem I have with this PR is that despite this being a good idea because it cleans up the test code and minimizes the number of DB accesses, which is always good, I believe that it might not solve the problem that happened in the other PR, as it seems weird that this was the problem.

My guess is a DB misconfiguration, because this is probably the first bit of test code that actually has to use the DB and thus the test times out. Probably something like that.

Finally, I think that the tests could be refactored a bit in terms of organization. The before and after hooks that are being changed here seem to only be needed for the final test inside this describe block.
I'd say that that calls for better organization of the tests, as they could be scoped like "Authentication tests" > "when a user already exists" > "something" or "Authentication tests" > "parameter validations/input validation" > "something".

More describe scopes might seem like clutter but might make the tests and test hooks more organized :slightly_smiling_face: